seatunnel-config/seatunnel-config-shade/src/test/java/org/apache/seatunnel/config/CompleteTest.java

```diff
@@ -40,14 +40,14 @@ public void testVariables() {
         systemProperties.put("city2", "shanghai");
 
         Config config = ConfigFactory
-            .parseFile(FileUtils.getFileFromResources("seatunnel/variables.conf"))
-            .resolveWith(ConfigFactory.parseMap(systemProperties), ConfigResolveOptions.defaults().setAllowUnresolved(true));
+                .parseFile(FileUtils.getFileFromResources("seatunnel/variables.conf"))
+                .resolveWith(ConfigFactory.parseMap(systemProperties), ConfigResolveOptions.defaults().setAllowUnresolved(true));
+
```

Comment:
Please remove these format, if there is no change of this file. Same with other test file.